### PR TITLE
cats: fix subscription view missing mariabackup plugin

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
- add missing mariabackup plugin
- increase catalog version to 2240
- fix issue #1759 

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
~~Required backport PRs have been created~~

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

**Review by sebsura — CHANGES_REQUESTED** [review]
So what about the new `bareos-fd-postgresql` plugin ?  Does this not get counted yet ?

File: core/src/cats/ddl/updates/postgresql.2230_2240.sql [hunk 1]
```diff
@@ -0,0 +1,126 @@
+-- update db schema from 2210 to 2230
+-- start transaction
+begin;
+
+drop index if exists file_jobid_idx;
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply core/src/cats/ddl/updates/postgresql.2230_2240.sql:5]: Yes you're right the explanation being not easy to understand.
I'm adding the why we do that now.
```
-- drop old index file_jobid_idx if still present.
-- replaced by file_jpfid_idx in 2014
-- PostgreSQL is able to use first column of partial index.
-- We can drop it as we only have PG backend.
```

File: core/src/cats/ddl/creates/postgresql.sql [hunk 2]
```diff
@@ -747,7 +750,7 @@ commit;
 -- Initialize Version
 --   DELETE should not be required,
 --   but prevents errors if create script is called multiple times
-DELETE FROM Version WHERE VersionId<=2230;
-INSERT INTO Version (VersionId) VALUES (2230);
+DELETE FROM Version WHERE VersionId<=2240;
+INSERT INTO Version (VersionId) VALUES (2240);
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply core/src/cats/ddl/creates/postgresql.sql:754]: Sorry I don't understand what is the `status` table ?

File: core/src/cats/ddl/updates/postgresql.2230_2240.sql [hunk 3]
```diff
@@ -0,0 +1,126 @@
+-- update db schema from 2210 to 2230
```
[the remark this answers is not in the document]
Comment (bruno-at-bareos) [reply core/src/cats/ddl/updates/postgresql.2230_2240.sql:1]: oh good catch. done

File: core/src/cats/ddl/creates/postgresql.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (sebsura) [reply core/src/cats/ddl/creates/postgresql.sql:754]: its a table that maps status codes to descriptions
```
CREATE TABLE Status (
    JobStatus         CHAR(1)     NOT NULL,
    JobStatusLong     TEXT,
    Severity          INTEGER,
    PRIMARY KEY (JobStatus)
);
```

File: core/src/cats/ddl/updates/postgresql.2230_2240.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (sebsura) [reply core/src/cats/ddl/updates/postgresql.2230_2240.sql:9]: Yep. Thats perfect.

**Review by sebsura — APPROVED** [review]
Good Work!

**Pull request merged**